src/components/NoteDialogComponents/NoteDialogHeader.scss

```diff
@@ -3,11 +3,32 @@
 .note-dialog__header {
   font-size: calc(24px + 0.8vw);
   color: $color-white;
-  width: max-content;
-  border-bottom: solid 3px var(--accent-color);
-  padding-bottom: $padding--default;
-  margin: $margin--default 0 $margin--large $margin--default;
+  width: 100%;
+  margin-bottom: $margin--large;
   letter-spacing: $letter-spacing--large;
   line-height: $line-height--large;
+  text-align: center;
   cursor: initial;
+  display: flex;
+  justify-content: center;
+
+  h2 {
+    width: max-content;
+    margin: 0;
+
+    &::after {
+      content: " ";
+      display: block;
+      height: 6px;
```

Comment:
The height refers to the blue part of the pseudo element. If I remove it, the blue line below the header disappears, because the element then has a height of 0

Reply:
Ah okay :D then this is fine, was just confused because I didn't see it. But I didn't try to change it either.